### PR TITLE
20.10.8: warning about $HTTP_PROXY breaking change

### PR DESCRIPTION
**Pull request by thomasgl-orange**
@thaJeztah: following up on https://github.com/moby/moby/pull/42643#issuecomment-892551186

### CODE REVIEW

File: engine/release-notes/index.md [hunk 1]
```diff
@@ -25,6 +25,10 @@ for Docker Engine.
 ## 20.10.8
 2021-08-03
 
+**IMPORTANT**: Due to [net/http changes](https://github.com/golang/go/issues/40909) in [Go 1.16](https://golang.org/doc/go1.16#net/http),
+setting an `$HTTP_PROXY` environment variable alone is not enough any more to properly configure an HTTP proxy.
+Make sure you also set an `$HTTPS_PROXY` environment variable, for handling requests to `https://` URLs.
+
```
Comment (thaJeztah) [inline engine/release-notes/index.md:31]: Thanks! I had a quick go at slightly reformatting the block to be rendered as a "note" (to make it stand out better), and adding a link the the relevant section in the docs on configuring a proxy.

```suggestion
> **IMPORTANT**
>
> Due to [net/http changes](https://github.com/golang/go/issues/40909) in [Go 1.16](https://golang.org/doc/go1.16#net/http),
> HTTP proxies configured through the `$HTTP_PROXY` environment variable are no
> longer used for TLS (`https://`) connections. Make sure you also set an `$HTTPS_PROXY`
> environment variable for handling requests to `https://` URLs.
>
> Refer to the [HTTP/HTTPS proxy section](../../config/daemon/systemd.md#httphttps-proxy)
> to learn how to configure the Docker Daemon to use a proxy server.
```

With the above suggestion applied, it looks like:

Screenshot 2021-08-04 at 13 54 22

Screenshot 2021-08-04 at 13 54 36

Comment (usha-mandya) [reply engine/release-notes/index.md:31]: Thanks @thaJeztah. Should we also use `{: .important }` as we have an admonition defined for Important notes. Also, can we change the uppercase IMPORTANT to Important?

Comment (thaJeztah) [reply engine/release-notes/index.md:31]: Gave it a quick go, and works for me! I updated the suggestion

Screenshot 2021-08-04 at 14 05 54
Screenshot 2021-08-04 at 14 06 01

**Review by usha-mandya — APPROVED** [review]
LGTM

**Pull request merged**